### PR TITLE
Fix/github1286

### PR DESCRIPTION
**Pull request by greglandrum**
Fix the problem by adding a member in the EmbeddedAtom class to flag atoms that should not be moved and then updating the code that tries to remove collisions to honor that flag.

### CODE REVIEW

File: Code/GraphMol/Depictor/EmbeddedFrag.cpp [hunk 1]
```diff
@@ -1677,16 +1680,36 @@ void EmbeddedFrag::flipAboutBond(unsigned int bondId, bool flipEnd) {
   endSideAids.clear();
   _recurseAtomOneSide(endAid, begAid, dp_mol, endSideAids);
 
-  // noew we have the molecule split into two groups of atoms
+  // look for fixed atoms in the fragment:
+  unsigned int nEndAtomsFixed = 0;
```
Comment (bp-kelley) [inline Code/GraphMol/Depictor/EmbeddedFrag.cpp]: Could this loop also be in the allIn loop or does recurseAtomOneSide modify d_eatoms?

File: Code/GraphMol/Depictor/EmbeddedFrag.cpp [hunk 2]
```diff
@@ -1759,18 +1782,20 @@ void EmbeddedFrag::openAngles(const double *dmat, unsigned int aid1,
...
   unsigned int aidB;
   RDKit::ROMol::ADJ_ITER nbrIdx, endNbrs;
   int type = 0;
-  if ((deg1 == 1) && (deg2 == 1)) {
+  if ((deg1 == 1 && !fixed1) && (deg2 == 1 && !fixed2)) {
     aidA = _findDeg1Neighbor(dp_mol, aid1);
     aidB = _findDeg1Neighbor(dp_mol, aid2);
     type = 1;
-  } else if ((deg1 == 1) && (deg2 > 1)) {
+  } else if ((deg1 == 1 && !fixed1) && (deg2 > 1 || fixed1)) {
```
Comment (bp-kelley) [inline Code/GraphMol/Depictor/EmbeddedFrag.cpp]: Is this logic correct?  deg2 >1 || fixed1 at the end looks suspicious.

Comment (greglandrum) [inline Code/GraphMol/Depictor/EmbeddedFrag.cpp]: I think you're right. Though it's going to take me a bit of staring at it to be sure.

File: Code/GraphMol/Depictor/EmbeddedFrag.cpp (hunk 1 above)
Comment (greglandrum) [inline Code/GraphMol/Depictor/EmbeddedFrag.cpp]: "allin loop"?

Comment (bp-kelley) [inline Code/GraphMol/Depictor/EmbeddedFrag.cpp]: The loop above that checks to see if the indices are "allIn".  That's the variable name.  This is more a curiosity than anything else.

**Comment by bp-kelley** [comment]
@greglandrum Just waiting for a comment, this code is currently a bit beyond me to suggest a test right now.

**Comment by bp-kelley** [comment]
No idea why I closed this, sorry.

File: Code/GraphMol/Depictor/EmbeddedFrag.cpp (hunk 2 above)
Comment (greglandrum) [inline Code/GraphMol/Depictor/EmbeddedFrag.cpp]: yeah, that's a typo. Fix coming in a minute or two (once the tests finish running)

**Comment by greglandrum** [comment]
this should be ready to go now.

**Pull request merged**